BrowserKit/Sources/Common/Extensions/String+Punycode.swift

```diff
@@ -0,0 +1,193 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+private let base = 36
+private let tMin = 1
+private let tMax = 26
+private let initialBias = 72
+private let initialN: Int = 128 // 0x80
+private let delimiter: Character = "-" // '\x2D'
+private let prefixPunycode = "xn--"
+private let asciiPunycode = Array("abcdefghijklmnopqrstuvwxyz0123456789")
+
+extension String {
```

Comment:
I think this is duplicated code that also exists in Client, correct? Shouldn't we delete it these and only use BrowserKit now?

Reply:
Indeed.